README.md

```diff
@@ -36,9 +36,12 @@ Simple C library for the MEGA65
    cd mega65-libc
    cmake -DCMAKE_PREFIX_PATH=$HOME/llvm-mos -B build
    make
+   make install # install library files
    make test # if `xmega65` (Xemu) was in your path when running cmake
    ~~~
 
+Installed artifacts should be made visible to cmake's package search, e.g. by adding their path to CMAKE_PREFIX_PATH.
```

Comment:
What does "artifacts" refer to? Line 37 points the prefix path to the SDK installation so that the compiler is detected. Please also place CMAKE_PREFIX_PATH in code brackets, i.e. `CMAKE_PREFIX_PATH`.

Reply:
Path of installed mega65-lib should be made visible to cmake, otherwise it wouldn't work, at least for me. Therefore installation should be stated as an explicit step. I've made another commit, hope it looks less confusing now :-)